app/event_source/subscribers/event_log_subscriber.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Subscribers
+  # Subscriber will receive Audit Log events
+  class EventLogSubscriber
+    include ::EventSource::Subscriber[amqp: 'enroll.event_log.events']
+
+    subscribe(
+      :on_enroll_event_log_events
+    ) do |delivery_info, _metadata, response|
+      logger.info '-' * 100 unless Rails.env.test?
+
+      payload = JSON.parse(response, symbolize_names: true)
+
+      subscriber_logger =
+        Logger.new(
+          "#{Rails.root}/log/on_enroll_event_log_events_#{TimeKeeper.date_of_record.strftime('%Y_%m_%d')}.log"
```

Comment:
is there a reason to create a custom log file and not use the standard log file ?

Reply:
but that didn't make it more difficult to capture with greylog ?